### PR TITLE
Some Nepali Translations

### PR DESCRIPTION
**Pull request by neogeomat**

### CODE REVIEW

**Comment by wondie** [comment]
Thank you very much Amirt! We look forward for your contirbution on STDM Transifex page and other parts of the STDM code!

File: i18n/stdm_ne.ts [hunk 1]
```diff
@@ -0,0 +1,7103 @@
...
+    <name>DTimeProperty</name>
+    <message>
+        <location filename="../ui/wizard/ui_dtime_property.ui" line="14"/>
+        <source>Datetime properties</source>
+        <translation>خصائص الوقت والتاريخ</translation>
+    </message>
+    <message>
+        <location filename="../ui/wizard/ui_dtime_property.ui" line="20"/>
+        <source>Minimum</source>
+        <
```
Comment (biplovbhandari) [inline i18n/stdm_ne.ts]: left out translation from Arabic. need to clean up before merging in trunk @neogeomat @wondie

Comment (wondie) [reply i18n/stdm_ne.ts]: That is right @biplovbhandari For STDM translation, we use Transifex ( https://www.transifex.com/stdm/stdm/dashboard/  ). We have already reached 70%+. However, for the Nepali translation to be incorporated on STDM, the translation of QGIS Desktop should increase at least to 20% ( https://www.transifex.com/qgis/QGIS/ ). Your contribution to both projects is much welcome! @neogeomat has already joined both projects.

Comment (biplovbhandari) [reply i18n/stdm_ne.ts]: Thanks for the instruction @wondie. I will look into that. Hope the training went well!

Comment (wondie) [reply i18n/stdm_ne.ts]: @biplovbhandari it went well! I will actually share a document on how to do it, which is for non-programmers. 
@neogeomat could you please send me a test email to my email wondim81@gmail.com I will send you the instruction too.

**Pull request closed without merging**